This patch removes the WIP from the version string. Upstream forgot to do that
for the release.
Index: debian.git/SConstruct
===================================================================
--- debian.git.orig/SConstruct	2009-08-25 17:31:31.803305643 +0200
+++ debian.git/SConstruct	2009-08-25 17:31:34.762806131 +0200
@@ -1,4 +1,4 @@
-env = AbracaEnvironment(APPNAME = 'abraca', VERSION = '0.4-WiP')
+env = AbracaEnvironment(APPNAME = 'abraca', VERSION = '0.4')
 
 env.VariantDir('build', '.')
 
